atom/browser/api/atom_api_system_preferences_mac.mm

```diff
@@ -28,6 +28,28 @@
 
 }  // namespace
 
+void SystemPreferences::PostNotification(const std::string& name, 
+    const base::DictionaryValue& user_info) {
+  DoPostNotification(name, user_info, false);
+}
+
+void SystemPreferences::PostLocalNotification(const std::string& name, 
+    const base::DictionaryValue& user_info) {
+  DoPostNotification(name, user_info, true);
+}
+
+void SystemPreferences::DoPostNotification(const std::string& name, 
```

Comment:
Do you need this helper method, if the implementation is different depending on the is_local flag?